docs/image-transfer-readme.md

```diff
@@ -0,0 +1,58 @@
+# Manual Workflow to Transfer Docker Images
+
+This workflow is designed to automate the process of transferring Docker images to a specified Docker Hub destination organization.
+It uses GitHub Actions to facilitate the image transfer.
+
+## Purpose
+The purpose of this workflow is to perform the following tasks:
+
+1. Update the configuration file config.yml with Docker Hub credentials and the destination organization. 
```

Comment:
Update the configuration file `config.yml` with Docker Hub credentials and the destination organization.